### ISSUE TITLE
Incorrect count of secondaries in Replset.py

### ISSUE THREAD

**Issue by bnogas** [Percona-Lab/mongodb_consistent_backup#56]
In this line secondary is counted as a replSet member only if it has better score than the previous one. 
https://github.com/Percona-Lab/mongodb_consistent_backup/blob/master/MongoBackup/Replset.py#L113

If there is more than three members in replSet it will randomly raise  "Not enough secondaries in replset %s to safely take backup!"

I don't know if it's intended for secondaries with too big lag to be counted to QUORUM(https://github.com/Percona-Lab/mongodb_consistent_backup/blob/master/MongoBackup/Replset.py#L119) or not (https://github.com/Percona-Lab/mongodb_consistent_backup/blob/master/MongoBackup/Replset.py#L109)

**Comment by Bastian82** [comment]
At [PerconaBlog](https://www.percona.com/blog/2017/01/09/mongodb-pit-backups-part-2/) i was asked to paste logs, so here you go:

```
[2016-11-08 12:17:02,395] [INFO] [MainProcess] [Backup:run:220] Starting mongodb-consistent-backup version 0.3.1 (git commit hash: 5acdc83a924afb017c0c6eb740a0fd7c2c0df3f6)
[2016-11-08 12:17:02,395] [INFO] [MainProcess] [Backup:run:230] Running backup of mP0104.dc5.alledc.net:27617 in replset mode
[2016-11-08 12:17:02,400] [INFO] [MainProcess] [Replset:find_primary:72] Found PRIMARY: dataB06-rs0/mC2475.dc4.local:27617 with optime Timestamp(1478603820, 6)
[2016-11-08 12:17:02,400] [INFO] [MainProcess] [Replset:find_secondary:128] Found SECONDARY dataB06-rs0/mP1818.dc5.alledc.net:27617: {'priority': 1, 'lag': -2, 'optime': Timestamp(1478603822, 2), 'score': 102}
[2016-11-08 12:17:02,401] [INFO] [MainProcess] [Replset:find_primary:72] Found PRIMARY: dataB06-rs0/mC2475.dc4.local:27617 with optime Timestamp(1478603820, 6)
[2016-11-08 12:17:02,401] [INFO] [MainProcess] [Replset:find_secondary:128] Found SECONDARY dataB06-rs0/mC1875.dc4.local:27617: {'priority': 1, 'lag': 0, 'optime': Timestamp(1478603820, 6), 'score': 97}
[2016-11-08 12:17:02,402] [INFO] [MainProcess] [Replset:find_primary:72] Found PRIMARY: dataB06-rs0/mC2475.dc4.local:27617 with optime Timestamp(1478603820, 6)
[2016-11-08 12:17:02,402] [INFO] [MainProcess] [Replset:find_secondary:128] Found SECONDARY dataB06-rs0/mP0104.dc5.alledc.net:27617: {'priority': 1, 'lag': -2, 'optime': Timestamp(1478603822, 4), 'score': 102}
[2016-11-08 12:17:02,402] [ERROR] [MainProcess] [Replset:find_secondary:134] Not enough secondaries in replset dataB06-rs0 to take backup! Num replset members: 2, required quorum: 3
[2016-11-08 12:17:02,402] [ERROR] [MainProcess] [Backup:exception:199] Problem getting shard secondaries! Error: Not enough secondaries in replset dataB06-rs0 to safely take backup!
Traceback (most recent call last):
  File "/root/.pex/install/MongoBackup-0.3.1-py2-none-any.whl.e86af08834b1b01d5e7ebcfb16f5a03333bfbf6e/MongoBackup-0.3.1-py2-none-any.whl/MongoBackup/Backup.py", line 243, in run
    secondary    = self.replset.find_secondary()
  File "/root/.pex/install/MongoBackup-0.3.1-py2-none-any.whl.e86af08834b1b01d5e7ebcfb16f5a03333bfbf6e/MongoBackup-0.3.1-py2-none-any.whl/MongoBackup/Replset.py", line 136, in find_secondary
    raise Exception, "Not enough secondaries in replset %s to safely take backup!" % rs_name, None
Exception: Not enough secondaries in replset dataB06-rs0 to safely take backup!
[2016-11-08 12:17:02,403] [INFO] [MainProcess] [Backup:cleanup_and_exit:171] Starting cleanup and exit procedure! Killing running threads
[2016-11-08 12:17:02,403] [INFO] [MainProcess] [Backup:cleanup_and_exit:194] Cleanup complete. Exiting
```

**Comment by timvaillancourt** [comment]
We believe this is resolved in our new 1.0.0 release. Please try this and reopen if this is not the case.

https://github.com/Percona-Lab/mongodb_consistent_backup/releases/tag/1.0.0.
